heron/config/src/yaml/conf/nomad/client.yaml

```diff
@@ -2,3 +2,7 @@
 heron.package.core.uri:                      file://${HERON_DIST}/heron-core.tar.gz
```

Comment:
Do we need this - if we are changing the config later in the same file?

Reply:
we are not changing it in the file.  The later config is commented an serves just as a reminder that user can do that as well